### PR TITLE
`txn-builder`: appoint-oracle

### LINKED ISSUE #244: Light wallet or third party dev friendly txn builder to generate signed transaction
Continue from #237 , extend the implementation for every required DfTx (prioritize light wallet requirement only)

Built [transaction](https://github.com/DeFiCh/jellyfish/blob/main/packages/jellyfish-transaction/src/tx.ts#L59) should be signed and ready for broadcast.

## CustomTx
- [x] utxosToAccount: `'U'` #245 
- [x] accountToUtxos: `'b'` #267 
- [x] accountToAccount: `'B'` #267 
- [x] addPoolLiquidity: `'l'` #268
- [x] removePoolLiquidity: `'r'` #268

## CustomTx (Oracles)
- [ ] appointOracle : `'o'` @monstrobishi 
- [ ] removeOracle: `'h'` @monstrobishi 
- [ ] updateOracle: `'t'` @monstrobishi 
- [ ] setOracleData: `'y'` @monstrobishi

### PR DESCRIPTION
**Pull request by monstrobishi**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Adds AppointOracle to txn-builder

#### Which issue(s) does this PR fixes?:

Fixes #244

### CODE REVIEW

File: packages/jellyfish-transaction-builder/src/txn/txn_builder_oracles.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { AppointOracle } from '@defichain/jellyfish-transaction/dist/script/defi/dftx_oracles'
+import { OP_CODES, Script, TransactionSegWit } from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+import { TxnBuilderError, TxnBuilderErrorType } from './txn_builder_error'
+
+export class TxnBuilderOracles extends P2WPKHTxnBuilder {
+  /**
+   * Requires UTXO in the same amount + fees to create a transaction.
```
Comment (fuxingloh) [inline packages/jellyfish-transaction-builder/src/txn/txn_builder_oracles.ts:8]:
```suggestion
```

Comment (fuxingloh) [reply packages/jellyfish-transaction-builder/src/txn/txn_builder_oracles.ts:8]: This description seems invalid

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_appoint_oracle.test.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+})
+
+describe('appoint oracle', () => {
+  beforeEach(async () => {
+    await container.waitForWalletBalanceGTE(1)
+  })
+
+  afterEach(async () => {
+    await container.waitForWalletBalanceGTE(1)
+  })
```
Comment (fuxingloh) [inline packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_appoint_oracle.test.ts:35]:
```suggestion
```

not required

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: packages/jellyfish-transaction-builder/src/txn/txn_builder_oracles.ts (hunk 1 above)
Comment (monstrobishi) [reply packages/jellyfish-transaction-builder/src/txn/txn_builder_oracles.ts:8]: oh yes I fixed in #307, will bring the change over

**Review by fuxingloh — APPROVED** [review]

**Review by canonbrother — APPROVED** [review]

**Pull request merged**